### DISCUSSION TITLE
do not show right in ui.scroll_area if wide bigger than 100

### DISCUSSION THREAD

**Discussion by tangshi00686** [zauberzeug/nicegui#1432] [Q&A]
### Question

```py
with ui.card().classes('w-100 h-100'):
    with ui.scroll_area():
```

as the code show above, if wide and height bigger than 100， it doesn't show right

**Comment by falkoschindler** [comment]
`w-100` and `h-100` are no valid Tailwinds classes.
Reference: https://tailwindcss.com/docs/width, https://tailwindcss.com/docs/height

You can either use Tailwind with custom dimensions like `w-[100px]` or use CSS: `.style('width: 100px')`.
